build.gradle

```diff
@@ -9,6 +12,11 @@ Base directory: $projectDir
 Running script ${relativePath(buildFile)}
 """
 
+if (file("vendor/protobuf").list().length == 0) {
+	throw new GradleException(
```

Comment:
That sounds good, but it that would update existing code too, we may want to do it a separate commit. We don't want to mix reformatting and real change in the same commit.